config/tsconfig.base.json

```diff
@@ -7,12 +7,12 @@
     "declaration": true,
     "declarationMap": true,
     "emitDeclarationOnly": true,
-    "module": "es2015",
+    "module": "CommonJS",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "strictNullChecks": true,
     "removeComments": false,
-    "target": "es2015",
+    "target": "ESNext",
```

Comment:
don't we want to make sure the target matches our lowest version of node?

Reply:
hmm, yeah, probably.  I'm not sure what that is.  I'll have to figure it out.

Reply:
OK, this is ES2020 for Node.js v14.17.0 as per [the kangax table](https://kangax.github.io/compat-table/es2016plus/)